src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -57,28 +61,62 @@ private void InstrumentModule()
                 {
                     foreach (var type in module.GetTypes())
                     {
-                        if (type.CustomAttributes.Any(a => a.AttributeType.Name == "ExcludeFromCoverageAttribute" || a.AttributeType.Name == "ExcludeFromCoverage"))
-                            continue;
-
-                        foreach (var method in type.Methods)
-                        {
-	                        var sourceFile = method.DebugInformation.SequencePoints.Select(s => s.Document.Url).FirstOrDefault();
-	                        if (!string.IsNullOrEmpty(sourceFile) && _excludedFiles.Contains(sourceFile)) {
-	                            continue;
-	                        }
-                            if (!method.CustomAttributes.Any(a => a.AttributeType.Name == "ExcludeFromCoverageAttribute" || a.AttributeType.Name == "ExcludeFromCoverage"))
-                                InstrumentMethod(method);
-                        }
+                        InstrumentType(type);
                     }
 
                     module.Write(stream);
                 }
             }
         }
 
+        private void InstrumentType(TypeDefinition type)
+        {
+            if (type.CustomAttributes.Any(IsExcludeAttribute))
+                return;
+
+            foreach (var method in type.Methods)
+            {
+                if (!method.CustomAttributes.Any(IsExcludeAttribute))
+                    InstrumentMethod(method);
+            }
+
+            foreach (var property in type.Properties)
```

Comment:
The C# compiler generates methods for properties. Those methods are returned as part of `type.Methods`, no need to explicitly instrument properties

Reply:
Thanks. You're right. I've removed enumeration of property methods.